src/DynamoCoreWpf/Extensions/ViewLoadedParams.cs

```diff
@@ -86,7 +86,7 @@ public Window DynamoWindow
             DynamoSelection.Instance.Selection.CollectionChanged += OnSelectionCollectionChanged;
         }
 
-        [Obsolete("Method will be deprecated in Dynamo 3.0, please use AddExtensionMenuItem")]
+        [Obsolete("Method will be deprecated in a future version of Dynamo, please use AddExtensionMenuItem")]
```

Comment:
who uses this?

Reply:
Again, I had failing tests that was using a test extension. So my thinking was that this might break more packages than we think? I can go back and fix the test package if you don't think that is the case.

Reply:
I mean, it will break the extension... but it's been obsolete for a while, hopefully our package authors are better at following our advice than we are...

defer to you.